server/src/test/java/org/apache/druid/server/coordinator/balancer/CostBalancerStrategyTest.java

```diff
@@ -334,42 +367,10 @@ public void testFindServerAfterExecutorShutdownThrowsException()
     );
   }
 
-  @Test(timeout = 90_000L)
-  public void testFindServerRaisesAlertOnTimeout()
+  private void verifyPlacementCost(DataSegment segment, ServerHolder server, double expectedCost)
```

Comment:
Why do we manually pass the expectedCost? 
Can we not compare the results of the method `computePlacementCost` with the sum of individual costs of the candidate segment across all segments on the server (i.e the original implementation)?

Reply:
Sure, that can be another assertion in this test or another test.

This particular test was added to ensure that cost remains the same even when we make changes to the way cost is computed, such as in this PR. That is why the exact numerical literals have been used.